.github/workflows/ci-cd.yml

```diff
@@ -24,35 +24,9 @@ env:
 
 jobs:
   build:
-    name: Build distribution 📦
-    runs-on: ubuntu-24.04
-    timeout-minutes: 5
-
-    steps:
-    - uses: actions/checkout@v4
-      with:
-        persist-credentials: false
-    - name: Install uv
-      # yamllint disable-line rule:line-length
-      uses: astral-sh/setup-uv@c7f87aa956e4c323abf06d5dec078e358f6b4d04  # v6.0.0
-      with:
-        # yamllint disable-line rule:line-length
-        enable-cache: | # zizmor: ignore[cache-poisoning] cache is disabled when publishing to prevent poisoning
-          ${{ github.ref_type == 'tag' && 'false' || 'auto' }}
-    - name: Build distribution 📦
-      run: uv build
-    - name: Check distribution 📦
-      run: uvx twine check --strict dist/*
-    - name: Upload distribution 📦
-      uses: actions/upload-artifact@v4
-      with:
-        name: python-package-distributions
-        path: dist/
+    uses: ./.github/workflows/reusable-build.yml
 
   test:
-    name: Test Python ${{ matrix.python-version }} on ${{ matrix.os }}
```

Comment:
@jakob-keller I missed this earlier, but the name should be set statically. Something like
```yaml
    name: 🧪 Tests${{ '' }}  # nest jobs under the same sidebar category
```
https://github.com/aio-libs/aiosignal/blob/0d5691b/.github/workflows/ci-cd.yml#L232

This will make sure that all the matrix jobs are grouped together in the left sidebar of the CI logs view. Currently, each of the matrix jobs is nested under a dedicated category, unnecessarily.

Reply:
#1354 addresses this